lib/object-adapter.js

```diff
@@ -1,12 +1,35 @@
-var ObjectAdapter = function() {};
-ObjectAdapter.prototype.build = function(Model, props) {
-  return props;
-};
-ObjectAdapter.prototype.save = function(doc, Model, cb) {
-  process.nextTick(function() {
-    cb(null, doc);
-  });
-};
-ObjectAdapter.prototype.destroy = function(doc, Model, cb) {};
-
-module.exports = ObjectAdapter;
+(function(adapter) {
+
+  if (typeof exports !== 'undefined') {
+    var ObjectAdapter = adapter();
+    if (typeof module !== 'undefined' && module.exports) {
+      exports = module.exports = ObjectAdapter;
+    }
+    exports.ObjectAdapter = ObjectAdapter;
```

Comment:
I used this as example [Underscore AMD](https://github.com/amdjs/underscore/blob/master/underscore.js)
Lines 58:65